tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+	"include": [
+		"types"
+	],
+	"compilerOptions": {
+		"resolveJsonModule": true,
+		"esModuleInterop": true,
+		"strict": true,
+		"noUnusedLocals": true,
+		"noUnusedParameters": true,
+		"noImplicitReturns": true,
+		"newLine": "lf",
+		"forceConsistentCasingInFileNames": true,
+		"pretty": true,
+		"target": "es5",
+		"lib": [
+			"dom",
+			"es2015"
```

Comment:
Definitely not a blocker for me but do we want this? I think we need to support ES5 directly in some repos like Vector.